example-application/api-under-test.js

```diff
@@ -82,20 +81,22 @@ const defineRoutes = (expressApp) => {
   expressApp.use("/order", router);
 
   expressApp.use(async (err, req, res, next) => {
-    console.log(err);
-    if (process.env.SEND_MAILS === "true") {
-      // important notification logic here
-      await mailer.send("Error", err.message, "admin@app.com");
-
-      // Other important notification logic here
+    if (typeof err === "object") {
+      if (err.isTrusted === undefined || err.isTrusted === null) {
+        err.isTrusted = true; //Error during a specific request is usually not catastrophic and should not lead to process exit
+      }
     }
+    await errorHandler.handleError(err);
     res.status(500).end();
   });
 };
 
-process.on("uncaughtException", () => {
-  // a log of other logic here
-  console.log("Error occured!");
+process.on("uncaughtException", (error) => {
```

Comment:
NIT: jsut for the uniformity
```suggestion
process.on("uncaughtException", (err) => {
```

Reply:
Uniformity with what? I usually prefer to use the entire word:

https://github.com/kettanaito/naming-cheatsheet#avoid-contractions

Reply:
With line `83` for example.
but, it's NIT.
and I'm ok with both `err` and `error`, we just choose one of them

Reply:
Gotcha, chose error and fixed